03/app.js

```diff
@@ -1,5 +1,22 @@
 document.addEventListener('DOMContentLoaded', init);
+const promise = fetch('https://api.ipify.org?format=json');
```

Comment:
Lepiej to zrobić wew. `click` bo chcemy odpytywać dopiero po kliknięciu, a w między czasie może zadziać się wiele rzeczy np. zmiana dostępu do Internetu i dane będą nieprawidłowe ;)